### PR TITLE
[ISSUE #310] add github action for check license

### PR DESCRIPTION
**Pull request by liangyuanpeng**
fixed #301

### CODE REVIEW

File: .github/workflows/check-license.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ This workflow will build a Java project with Maven
```
Comment (kezhenxu94) [inline .github/workflows/check-license.yml:1]: This line is wrong in syntax. Should start with #

Comment (liangyuanpeng) [reply .github/workflows/check-license.yml]: Thanks for your review.

**Comment by SteveYurongSu** [comment]
Hi, will you continue to work on this PR?

If you are no longer working on this PR, I would like to continue to maintain your existing work (by pulling your PR) and submit a new PR (then you will be a co-author), thanks!

**Comment by liangyuanpeng** [comment]
So, This PR have any problem? I juse waiting for review and got nothing.

**Comment by qqeasonchen** [comment]
> Hi @liangyuanpeng , will you continue to work on this PR?
> 
> If you are no longer working on this PR, I would like to continue to maintain your existing work (by pulling your PR) and submit a new PR (then you will be a co-author), thanks!

@liangyuanpeng I thought @SteveYurongSu will solve the probmen in #377 , my concern is still exists.

**Pull request merged**